example/spec/parallel/syncGranule/SyncGranuleSuccessSpec.js

```diff
@@ -416,6 +421,59 @@ describe('The Sync Granules workflow', () => {
     });
   });
 
+  describe('when an ACL is provided in the task config and set to the canned ACL authenticated-read', () => {
+    const meta = { ACL: 'authenticated-read' };
```

Comment:
So given the change to only allow `private` and `disabled`, I'm not totally sure how this is passing. I think I just don't understand the expected outputs, e.g. `FULL_CONTROL`.

It seems like the new logic in `setOrDisableAcl` should set this to `private`. Does `private` then result in `FULL_CONTROL` and `authenticated-read`? 

I could be misunderstanding what's happening.

Reply:
Yeah I tried to explain when I opened the PR but the somewhat unfortunate thing about the `getObjectAcl` function is that it only returns the permissions and not the actual ACL of the object. And in many (perhaps all) cases for canned ACLs, the permissions look the same (aka FULL_CONTROL) which is probably why this passes. Let me remove this test.

Reply:
Ah, so the `aunthenticated-read` ACL will have a `FULL_CONTROL` permission, like you said, but the lamdaOutput will still contain `authenticated-read` since that was what was inputted for `meta.ACL` 🤔  Which is why this test passes. The tricky thing is, it would be kind of hard to determine the accuracy of the test given that both `authenticated-read` and `private` will return `FULL_CONTROL` permissions for `getObjectAcl` 😢  

I can remove this test as well if we feel the unit test sufficiently captures the case that a user tries to input something other than `disabled`

Reply:
Cool. Testing the case where someone enters something other than disabled in units and not integration feels right to me I think. Or clarifying what's going on here in some way, but I think you can probably just remove this test.